docs/source/changelog.md

```diff
@@ -12,6 +12,326 @@ and as we merge [breaking changes in pull
 requests](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pulls?q=is%3Apr+is%3Aclosed+label%3Abreaking),
 this list should be updated.
 
+## 2.0
+
+### 2.0.0
+
+#### Highlights
+
+- **TODO**
+
+  TODO
```

Comment:
What else should we highlight? It seems that most of the information below before the PR list _is_ the highlights, so maybe we should omit this? Or increase level so everything is h5 under the highlights h4?

Reply:
I think a short paragraph rather than just bullet points would be nice to have here. I'll work on it unless someone else gets there first.